### PR TITLE
Allow adding custom class to the main modal element

### PR DESCRIPTION
**Pull request by dminchev**
This feature allows users to add custom classes to the master modal element. It is useful when you have customized bootstrap theme which has specific modal behaviors (like the way they popup).

### CODE REVIEW

File: vendor/assets/javascripts/data-confirm-modal.js [hunk 1]
```diff
@@ -119,9 +120,10 @@
   var buildModal = function (options) {
     var id = 'confirm-modal-' + String(Math.random()).slice(2, -1);
     var fade = settings.fade ? 'fade' : '';
+    var customClass = settings.customClass ? customClass : '';
```
Comment (vjt) [inline vendor/assets/javascripts/data-confirm-modal.js]: this should read

var customClass = settings.customClass || ''

**Comment by vjt** [comment]
useful, thanks. I would rename customClass in modalClass, though.

**Comment by dminchev** [comment]
Welcome, just added a small fix, because it was missing the settings. prefix

**Comment by vjt** [comment]
Thanks

**Pull request merged**